src/accapi/structs.py

```diff
@@ -201,39 +229,46 @@ def __init__(self, *args):
         self.trackName = args.pop(0)
         self.trackId = args.pop(0)
         self.trackMeters = args.pop(0)
-        self.cameraSets = {args.pop(0): [args.pop(0) for _ in range(args.pop(0))] for _ in range(args.pop(0))}
+        d = {}
```

Comment:
Points granted for readability, not sure which backwards compatibility feature is broken here however. In any case, I've made this even more readable (and hopefully also backwards compatible) in #4.